### ISSUE TITLE
Reimporting txqueue on startup can be very noisy 

### ISSUE THREAD

**Issue by bohendo** [openethereum/openethereum#9789]
_Before filing a new issue, please **provide the following information**._

- **Parity Ethereum version**: 2.1.3-beta
- **Operating system**: Linux (Ubuntu 16.04 docker container)
- **Installation**: `curl https://get.parity.io -Lk > /tmp/get-parity.sh && bash /tmp/get-parity.sh`
- **Fully synchronized**: yes
- **Network**: ethereum
- **Restarted**: yes

My parity node has been synced with the network and serving light clients for many months. In these months, I've accumulated a staggering number of logs that inform me of local transactions that are "below the minimal gas price accepted."

These logs seem like they should be ephemeral but they persist between server restarts. I can't watch the logs when my server restarts otherwise so many of these spam messages will get printed that some buffer fills up & my terminal freezes.

The logs start at transaction hash `0x0000037d90a2e7c92ae83e49ec0d241d7a981d7eccec3a92d90e851fda36d7de` and print out in order until my terminal stops responding a minute or so later around tx hash `0x9..` or `0xa..`.

I don't send *that* many transactions so I assume all these "local" transaction logs are accumulating because I've been serving light clients.

The only solution I see now is to stop serving light clients and then delete/resync my blockchain data but I like serving light clients so would rather not go this route..

I'd much rather figure out where exactly these logs are being saved so I can delete only these logs & save myself the trouble of re-syncing.

Even better, I'd love it if these logs didn't persist between restarts. I'd also be perfectly happy if they weren't generated at all.

**Comment by joshua-mir** [comment]
You can specify the location of the log file with `--log-file` 

Please share that log file here. I don't understand what you mean by the log file is persisting, it sounds much more like you have logging enabled for your transaction pool, and *that* does persist between restarts (unless you use `--no-persistent-txqueue`)

**Comment by bohendo** [comment]
Glory hallelujah this was exactly what I was looking for.

I added the `--no-persistent-txqueue` flag, restarted, and this flood of logs completely disappeared.

Then I removed that flag (because I *would* actually like my tx queue to persist between restarts) and all these logs are still gone.

Looks like I'll just have to restart with this flag monthly or so to keep things under control & I'm good.

Although, I'd suggest that most users probably want their tx queue to persist & not their tx queue related logs. I'd be a happy camper if I received these logs in real time but didn't need to manually clear them every month.

**Comment by bohendo** [comment]
Glory hallelujah this was exactly what I was looking for.

I added the `--no-persistent-txqueue` flag, restarted, and this flood of logs completely disappeared.

Then I removed that flag (because I *would* actually like my tx queue to persist between restarts) and all these logs are still gone.

Looks like I'll just have to restart with this flag monthly or so to keep things under control & I'm good.

Although, I'd suggest that most users probably want their tx queue to persist & not their tx queue related logs. I'd be a happy camper if I received these logs in real time but didn't need to manually clear them every month.
